odata-parser.ometajs

```diff
@@ -573,9 +559,9 @@ export ometa ODataParser {
 
 	// TODO: This should really be done treating everything the same, but for now this hack should allow FF to work passably.
```

Comment:
NIT: Should we move this comment to the `IsApostrophe` function?

Reply:
I moved it